cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1963,7 +1963,7 @@ search = `cs_o365` _index_earliest=-61m@m _index_latest=-1m@m (sourcetype="o365:
 | stats latest(Status) as Status latest(_time) as _time by Service host \
 | search Status IN ("ExtendedRecovery","InformationUnavailable","Investigating*","RestoringService","ServiceDegradation","ServiceInterruption") \
 | `cs_human_readable_time_format(_time, LastUpdatedTime)` \
-| eval cyences_severity = "medium" \
+| eval cyences_severity = "low" \
```

Comment:
I have a concern from a security standspoint, is it really a low severity alert??

Reply:
We can discuss it in our Soc meeting today before merging